spec/system/health_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe 'Health check' do
+  it 'should check if the app is ok and connected to a database' do
```

Comment:
Rubocop doesn't like this and fails the pipeline…
```suggestion
RSpec.describe 'HealthController' do
  it 'returns the correct response if the app ok and connected to a database' do
```